src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -430,17 +442,17 @@ CrashInfo::GetBaseAddressFromAddress(uint64_t address)
 uint64_t
 CrashInfo::GetBaseAddressFromName(const char* moduleName)
 {
-    for (const ModuleInfo& moduleInfo : m_moduleInfos)
+    for (const ModuleInfo* moduleInfo : m_moduleInfos)
     {
-        std::string name = GetFileName(moduleInfo.ModuleName());
+        std::string name = GetFileName(moduleInfo->ModuleName());
 #ifdef __APPLE__
```

Comment:
The only thing is this is configurable, but this is not the default. I don't know how common it is to request case sensitivity.

Reply:
Lee mentioned it in passing and then I confirmed on my local MacBook.  Not sure what to do if it is configurable.  I'm going to leave it this way for now.